src/bundle/Resources/config/services.yml

```diff
@@ -64,6 +64,7 @@ services:
 
 
     EzSystems\EzPlatformAdminUi\Notification\FlashBagNotificationHandler:
+        lazy: true
```

Comment:
Would it be better to refactor `FlashBagNotificationHandler` to look something like https://github.com/ezsystems/ezplatform-admin-ui/pull/241, rather than relying on the service being lazy?